docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,54 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and we want the library content to be stored in the Blockstore.
+To achieve this we need a referencing mechanism to reference the libraries to the courses. This referencing helps in replacing
+v1 libraries (current content libraries stored on Modulestore) with v2 libraries (Blockstore backed) which are:
+
+#. not dependent on edx-platform
```

Comment:
Blockstore is being moved _into_ edx-platform so it is dependent on edx-platform. However, it is much more flexible than modulestore and can store anything that can be represented as a file - unlike modulestore, which only stores courses built of XBlocks.